portfolio/src/main/webapp/script.js

```diff
@@ -27,17 +27,21 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-// /** Redirects to the "Home" page. */
-// function goToHomePage() {
-//     location.href = 'index.html';
-// }
-
-// /** Redirects to the "Contact" page. */
-// function goToContactPage() {
-//     location.href = 'contact.html';
-// }
-
-// /** Redirects to the "Contact" page. */
-// function goToContactPage() {
-//     location.href = 'contact.html';
-// }
+// Get all comments and print them on index.html
+async function updateCommentSection() {
+    const response = await fetch('/data');
+    var commentList = await response.text();
+    
+    // Split the comment list in comments
+    commentList = commentList.slice(1, -2);
+    let comments = commentList.split(", ");
+    
+    // Add every comment to the DOM
+    var i, node, text;
+    for (i = 0; i < comments.length; i++) {
+        node = document.createElement('LI');
```

Comment:
Not sure here. But should this be lower case li to match the html tag?

Reply:
I did it this way because this was the way the tutorial was done.
I checked now and it seems that both ways work. I will modify it to lower case.

Reply:
it is not case sensitive uppercase looks antique though :-)